PowerEditor/src/DarkMode/DarkMode.ipp

```diff
@@ -0,0 +1,276 @@
+#pragma once
```

Comment:
Might be my experience with boost leaking in here; an .ipp is basically a header that is intended to be included in just one translation unit, rather than compiled on its own. However I am happy to rename it or move it into a classic .cpp and .h file instead.

Reply:
Now it is a separate .h and .cpp!